tests/yam/agama/agama.pm

```diff
@@ -0,0 +1,49 @@
+## Copyright 2023 SUSE LLC
```

Comment:
I would recommend another module name , like run_playwright_test.pm instead of agama.pm

Reply:
please, do not use the technology running behind for naming. Also use something meaningful from the user/QE perspective. /agama/agama.pm and /agama/auto.pm should suffice to understand ... most likely that "auto" could collide in name with something else, and perhaps should be "agama_auto.pm".

Reply:
What can someone understand by that name without knowing the content of the module? IMO the naming is very abstract.

Reply:
and why do we have a directory named agama and then also use agama_ prefix for every module?

Reply:
we just removed all the `agama_*` in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17629
we shouldn't have duplicated code in the first place, that is unfortunate, we need to keep good practice in mind.
instead of agama it could be called agama_interactive to distinguish from the the agama_auto, but it is not needed, because the default way we consider is the interactive (auto is probably what customer will try to use but it is the fancy way to do it, the mos obvious way is interactive). The thing is that we use openQA just to trigger this, so just one test module for agama we are fine most likely.